rpe/extractors/gcp_auditlogs.py

```diff
@@ -433,4 +425,22 @@ def add_resource():
             }
             add_resource()
 
+        elif (
```

Comment:
is this only for testing, dont think it is used anywhere else... we should probably move this to tests then? similar codebase exists in rte which makes me think why we have it in 2 places, this can be a separate cleanup, not something that this change has introduced..

Reply:
The AuditLog extractor is right now used only in vee-publisher, but this is a functionality of the engine, so for engine completeness, I added this resource. 

Yes, code is sort of duplicates in real-time-enforcer, and there should be a cleanup

Reply:
can rte also refer to auditlogextractor like vee?

Reply:
yes, I think so, but I think can be a part of a separate code cleanup

Reply:
can you open a jira for that I can close this comment..